lib/puppet/provider/zabbix.rb

```diff
@@ -49,7 +49,7 @@ def self.create_connection
     zbx = ZabbixApi.connect(
       url: "#{protocol}://#{api_config['default']['zabbix_url']}/api_jsonrpc.php",
       user: api_config['default']['zabbix_user'],
-      password: api_config['default']['zabbix_pass'],
+      password: "#{api_config['default']['zabbix_pass']}",
```

Comment:
I don't see why this would be needed. You're casting a string to another string.